pyproject.toml

```diff
@@ -21,10 +21,12 @@ classifiers = [
 include = [
     "CHANGELOG.md",
     "LICENSE",
+    "netutils/protocols.json",
 ]
 
 [tool.poetry.dependencies]
 python = "^3.6"
+setuptools = "^57.4.0"
```

Comment:
ohh man, I am the worst... we have left 0 dependencies, and would like to stay that way... sorry for doing this :(